src/Microsoft.DotNet.CodeAnalysis/PackageFiles/PinvokeAnalyzer.props

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hmm.. I did that initially however thought it might be better to put pinvoke specific props in a separate props file for better classification. Although I find this preferable, if you feel strongly about it, I will change it :). Let me know.